### PR TITLE
(Added)[JIRA-2804]-fastlane configuration to Integration tests (ExampleSwift) #2603

### PR DESCRIPTION
**Pull request by matheusctMeli**
https://mercadolibre.atlassian.net/browse/PXN-2804

I configured fastlane to ExampleSwift to build a .ipa and .app, upload and start integration tests in saucelabs

What have changed?
added fastlane folder on ExampleSwift (fastfile, gymfile, matchfile) changed the bundle identifier from ExampleSwift (com.mercadolibre.-) to build the ipa.

also modified some configurations on project ExampleSwift like environment and add accessibility identifier on fields to facilitate find filds in appium.
also in px lib add security code componente a accessibility identifier (without it appium can't access it)  and changed the productid for a value that provide more creditcards to use on tests.

Next step:
add a input on custom initialize to modifi the productid on demand in ExampleSwift

How to test:
using the workflow integration_tests on bitrise https://app.bitrise.io/app/d2d19a45654ed1d8#/builds
or build de .app using the lane build_local in ExampleSwift and execute the test using appium 

Prints for layout changes:
none

Extras:

### CODE REVIEW

File: ExampleSwift/fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  lane :build do
+    cocoapods(
+      clean_install: true,
+      podfile: "./Podfile"
+    )
+    match(app_identifier: "com.mercadolibre.*", type: "development", readonly: true)
+    gym
+    sh "bash ../scripts/saucelabs.sh"
+  end
+end
```
Comment (pedro-nadolny) [inline ExampleSwift/fastlane/Fastfile:34]: Nosso fastfile está em /.fastlane/Fastfile a partir da raiz do projeto. Da um fix so nesse path junto com o Gymfile e Matchfile antes de mergear. 🙏

**Review by ricardo-dalambert-meli — APPROVED** [review]

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The PR title should follow the title convetion: (Added,Changed,Deprecated,Removed,Fixed,Security)[JIRA-XXX] - (brief description here)</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR does not include any JIRA tasks in the title. (e.g. [TICKET-1234])</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">You and the size of your PR are awesome! 🚀</td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `3.15%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: MercadoPagoSDK/MercadoPagoSDK/Config/External/BiometricModule/PXBiometricConfig.swift [hunk 2]
```diff
@@ -29,7 +29,7 @@ extension PXBiometricConfig {
     }
 
     static func createConfig(withFlowIdentifier: String? = nil, andAmount: NSDecimalNumber? = nil) -> PXBiometricConfig {
-        let pxProductId: String = "BJEO9TFBF6RG01IIIOU0"
+        let pxProductId: String = "BJEO9NVBF6RG01IIIOTG"
```
Comment (ricardo-dalambert-meli) [inline MercadoPagoSDK/MercadoPagoSDK/Config/External/BiometricModule/PXBiometricConfig.swift:32]: Essa alteração era de fato esperada para essa entrega do fastlane?

File: MercadoPagoSDK/MercadoPagoSDK/UI/SecurityCode/PXSecurityCodeViewController.swift [hunk 3]
```diff
@@ -263,6 +263,7 @@ private extension PXSecurityCodeViewController {
         andesTextFieldCode = AndesTextFieldCode(label: viewModel.getAndesTextFieldCodeLabel(), helpLabel: nil, style: viewModel.getAndesTextFieldCodeStyle(), state: .IDLE)
         andesTextFieldCode.delegate = self
         andesTextFieldCode.alpha = 0
+        andesTextFieldCode.accessibilityIdentifier = "securityCode"
```
Comment (ricardo-dalambert-meli) [inline MercadoPagoSDK/MercadoPagoSDK/UI/SecurityCode/PXSecurityCodeViewController.swift:266]: Essa alteração era de fato esperada para essa entrega do fastlane?

File: MercadoPagoSDK/MercadoPagoSDK/RequestLayer/RequestStruct/Request.swift [hunk 4]
```diff
@@ -27,7 +27,7 @@ enum HeaderFields: String {
 
 final class Request<Target: RequestInfos>: RequestProtocol {
     // MARK: - Perivate properties
-    private let defaultProductId = "BJEO9TFBF6RG01IIIOU0"
+    private let defaultProductId = "BJEO9NVBF6RG01IIIOTG"
```
Comment (ricardo-dalambert-meli) [inline MercadoPagoSDK/MercadoPagoSDK/RequestLayer/RequestStruct/Request.swift:30]: Essa alteração era de fato esperada para essa entrega do fastlane?

**Review by melicocuroci — APPROVED** [review]
@matheusctMeli deixei minha aprovação, mas de uma olhada nos comentários do Ricardo antes de mergear.

File: MercadoPagoSDK/MercadoPagoSDK/UI/SecurityCode/PXSecurityCodeViewController.swift (hunk 3 above)
Comment (matheusctMeli) [reply MercadoPagoSDK/MercadoPagoSDK/UI/SecurityCode/PXSecurityCodeViewController.swift:266]: não para o fastlane, mas para toda a estoria de testes de integração

**Review by melicocuroci — APPROVED** [review]

**Pull request merged**